browser/ui/webui/settings/brave_default_extensions_handler.cc

```diff
@@ -31,8 +32,27 @@
 #include "extensions/browser/extension_system.h"
 #include "extensions/common/feature_switch.h"
 
+#if BUILDFLAG(ENABLE_TOR)
+#include "brave/browser/tor/tor_profile_service.h"
+#include "brave/common/tor/pref_names.h"
+#endif
+
+#if BUILDFLAG(ENABLE_TOR)
+namespace {
+// Initial value is stored to check whether option is changed or not.
```

Comment:
I missed this the first time and I'm not understanding what's going on here. How could `g_initial_tor_enabled_set` ever be `true` here? https://github.com/brave/brave-core/pull/4046/files#diff-799fb4107cdd8b67f67de3b96c117eb8R51

Reply:
`IsTorDisabled` is a pref value and if you want to know if it has been modified then you compare it to the default pref value `GetDefaultPrefValue`

Reply:
I think `Initial` made you confusing. It's not for getting default value.
I wanted to use that value to know whether tor enabling is changed or not after launching.

Reply:
These global variables are deleted and `TorProfileService::IsTorDisabledChanged()` is added.